SharedClasses/ConfigManager.cs

```diff
@@ -137,6 +137,15 @@ public static bool IsClientDebugModeEnabled()
         {
             return GetResourceMetadata("vMenu", "client_debug_mode", 0).ToLower() == "true";
         }
+        
+        /// <summary>
+        /// Default value for server-side chameleon colours.
+        /// </summary>
+        /// <returns></returns>
+        public static bool IsServerUsingChameleonColours()
+        {
+            return GetConvar("server_using_chameleon_colours", "false");
```

Comment:
This needs to be an option in the default permissions.cfg as well, and it should be set to `setr` so it's replicated for client side use.